react/modules/interpolateLink.ts

```diff
@@ -5,19 +5,24 @@ interface Params {
   namespace?: string
   context?: Record<string, any>
   contextType: AvailableContext
+  escapeLinkRegex?:RegExp
 }
 
 export default function interpolateLink(params: Params) {
-  const { link, namespace, context, contextType } = params
+  const { link, namespace, context, contextType , escapeLinkRegex } = params
```

Comment:
```suggestion
  const { link, namespace, context, contextType, escapeLinkRegex } = params
```

Reply:
This suggestion isn't what the lint would do, it will also break lines for each property. So, I suggest lint this file too